examples/documentation.html

```diff
@@ -327,7 +327,7 @@
 
         <div style="height: 50px"></div>
 
-		<h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>
+    <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist>ul').toggle()">toggle all</a> ]</span></h3>

```

Comment:
Looks to me like an unecessary commit and that this change messed up the indentation.